apps/data/lib/data/opportunities.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Data.Opportunities do
+  @moduledoc """
+  The public API for `opportunity` resources
+  """
+
+  alias Data.{Opportunity, Project, Repo}
+
+  def get(params), do: Repo.get_by(Opportunity, params)
+
+  def insert(params) do
+    %Opportunity{}
+    |> Opportunity.changeset(params)
+    |> Repo.insert
+  end
+
+  def update(id, params) when is_integer(id) do
+    Opportunity
+    |> Repo.get(id)
+    |> update(params)
+  end
+
+  def update(struct, params) do
```

Comment:
Since this isn't a library I'm not going to spend too much time on documentation, no one will be using these functions unless they're working in this particular project.  

The public APIs all have test coverage in the actual tests 👍 

If someone wants to add more documentation later that is certainly welcome, but I don't want it holding up the launch of the MVP.